frontend/src/components/PlayComponents/SelfCard.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components/macro';
+import ActionButton from '../commons/ActionButton';
+import Input from '../commons/Input';
+import { GiTwoCoins } from 'react-icons/gi';
+
+export default function SelfCard({ bet, fold, minBet, cash, active }) {
+  const [betValue, setBetValue] = useState(minBet);
+  const [betTooSmall, setBetTooSmall] = useState(false);
+  const [betToLarge, setBetTooLarge] = useState(false);
+
+  useEffect(() => {
+    setBetTooLarge(betValue > cash);
+    setBetTooSmall(betValue < minBet);
+  }, [betValue, cash, minBet]);
```

Comment:
```suggestion
  const betTooSmall = betValue > cash;
  const betTooLarge = betValue < cash;
```
You can derive these values form only one state.